---
layout: poster
title: "Uncovering imprinting of PAV genes and transposable elements using whole genome assemblies"
image: /images/posters/Higgins_maize20.pdf
authors: Higgins K, Zhou P, Brandvain Y, Springer N, Anderson SN
year: 2020
ref: Higgins Maize Meeting 2020
journal: Maize Genetics Conference 2020 [canceled due to COVID-19]
pdf: /images/posters/Higgins_maize20.pdf
doi: 
supplement: 
---

Genomic imprinting, or parent-of-origin biased gene expression, has been reported in the endosperm of many plant species. Imprinted genes exhibit preferential expression of the maternal or paternal allele and are often associated with allele-specific variation for DNA methylation or histone modifications. While some genes have conserved imprinted expression patterns across plant species, there are also examples of genes with imprinting status that varies across different haplotypes within the same species, leading to the question of how genetic variation and epigenetic variation interact to influence gene expression patterning. Assessing how genomic imprinting arises has been limited by the technical difficulties associated with calling allele-specific expression using a single reference genome where only loci that are shared between genotypes and contain SNPs that distinguish alleles can be assayed. To overcome this limitation, we performed RNA-seq on reciprocal crossed hybrid endosperm between maize genotypes with complete, de-novo genome assemblies. By mapping to whole genome assemblies of both parents and comparing expression across reciprocal crosses, we were able to identify imprinted expression of both shared and non-shared portions of the genome. This allowed the identification of imprinting for several genes with present-absent variation (PAV) among genotypes. This approach also enabled the analysis of imprinting at transposable elements. We find > 100 imprinted transposable elements, over half of which show presence/absence variation in genome content between genomes. Interestingly, while genes show both maternally-biased and paternally-biased patterns of imprinting, the vast majority of imprinted transposable elements exhibit maternally-biased expression. By analyzing examples variable imprinting across genotypes, we are beginning to define the mechanisms that underlie parent-specific expression patterns in the endosperm.
