---
layout: member
title: Sarah N Anderson
position: Postdoc
handle: sarahanderson
twitter: sna8
orcid: 0000-0002-1671-2286
calendar: 
github: 
linkedin:
email: sna@umn.edu
scholar: https://scholar.google.com/citations?user=https://scholar.google.com/citations?user=mlAcnHkAAAAJ&hl=en&user=f7FJ4UoAAAAJ
image: /images/people/sarah-anderson.jpg
cv: /pdfs/people/sarah-anderson-cv.pdf
---
A spectacular array of phenotypic variation exists in nature resulting from a combination of genetic, epigenetic, and environmental influences. My work aims to understand the genetic contribution to this diversity by moving beyond SNPs to understand how variation in transposable elements (TEs) contribute to a dynamic genome and transcriptome. Transposable elements contribute a large amount of DNA sequences to genomes and are a major source of genetic variation within species, particularly within large genomes. Most TE insertions are deleterious or at best neutral, but TEs are distinct from other common mutation types (SNPs and indels) in that they can introduce regulatory elements and alter the chromatin state of a region upon transposition. This means that TEs have a unique potential to create beneficial alleles and novel regulatory networks by altering expression of nearby genes.

A major interest of my research is understanding how TEs influence gene expression across development and in response to stress. Expression of a gene is influenced by a complex combination of proximal and distal regulatory elements and epigenetic modifications, and many genes show dynamic expression patterns that vary across time, cell types, and environments. Different genotypes within a species exhibit different transcriptome changes in response to stress, and stress resistance loci can have qualitative or quantitative effects on phenotypes. One major goal in modern plant biology is to create crops with improved resilience to abiotic and biotic stresses, and the ability to fine-tune transcriptional responses to stress in a quantitative manner would aid in creating more resilient plants without the yield penalty associated with constitutively overexpressing stress resistance genes. 

I am currently involved in several projects that broadly aim to understand the role of maize TEs in creating dynamic plant genomes, transcriptomes, and epigenomes. 
